aries_vcx_core/Cargo.toml

```diff
@@ -11,9 +11,12 @@ credx = ["dep:indy-credx"]
 vdr_proxy_ledger = ["credx", "dep:indy-vdr-proxy-client"]
 # Feature flag to allow legacy proof verification
 legacy_proof = []
+askar_tests = []
```

Comment:
Not only tests, but anything Askar related in the codebase should be behind a feature flag. Given the implementation must stay hidden behind the BaseWallet trait, the rest of the codebase should not know of Askar's existence.
You might be aware of this, but keeping that constraint will "reviewlessly" assure these desired properties.

Reply:
For example having it feature flagged from get go would probably naturally lead you to not create `Askar*` specific error kinds. It might be somewhat restrictive while POCing, but ultimately will guide you toward the optimal design/integration with the rest of the codebase